S2VX.Game.Tests/HeadlessTests/ScoreProcessorTests/ProcessHitCursorTests.cs

```diff
@@ -1,40 +1,34 @@
 ﻿using NUnit.Framework;
 using osu.Framework.Allocation;
-using osu.Framework.Audio;
-using osu.Framework.Screens;
 using osu.Framework.Testing;
-using S2VX.Game.Play;
 using S2VX.Game.Play.UserInterface;
 using S2VX.Game.Story;
 using S2VX.Game.Story.Note;
-using System.IO;
 
 namespace S2VX.Game.Tests.HeadlessTests.ScoreProcessorTests {
     [HeadlessTest]
     public class ProcessHitCursorTests : S2VXTestScene {
         [Resolved]
         private S2VXCursor Cursor { get; set; }
 
-        private PlayScreen PlayScreen { get; set; }
+        [Cached]
```

Comment:
I might've forgot, but why do we need cached here? I don't see where this would be resolved. Does it have something to do with dependencies?

Reply:
Tip for reviewing. Select a group of lines or start your review point lower because from the preview window, I can't see the variable you're talking about.

In this case, `Story` needs to be resolved by `ScoreProcessor`.

Reply:
Oh ok. That makes sense, thanks.